dist/yoast-seo.js

```diff
@@ -2297,21 +2307,36 @@ YoastSEO.SnippetPreview.prototype.getUnformattedText = function( ev ) {
 	}
 };
 
-YoastSEO.SnippetPreview.prototype.setUnformattedElemText = function( elem ) {
-	this.unformattedText[ elem ] = document.getElementById( elem ).textContent;
-};
-
 /**
  * when text is entered into the snippetPreview elements, the text is set in the unformattedText object.
  * This allows the visible data to be editted in the snippetPreview.
  * @param ev
  */
 YoastSEO.SnippetPreview.prototype.setUnformattedText = function( ev ) {
-	this.setUnformattedElemText ( ev.currentTarget.id );
+	var elem =  ev.currentTarget.id;
+	this.unformattedText[ elem ] = document.getElementById( elem ).textContent;
+};
+
+/**
+ * Adds the siteName to the snippetTitle when editting starts, this adds the sitename to
+ * the snippet_sitename that cannot be edited.
+ * @param ev
+ */
+YoastSEO.SnippetPreview.prototype.setSiteName = function( ev ) {
+	if ( ev.currentTarget.id === "snippet_title" ) {
+		document.getElementById( "snippet_sitename" ).textContent = this.snippetSuffix;
+	}
 };
 
 /**
- * adds and remove the tooLong class when a text is too long.
+ * Removes the siteName from the snippetTitle when editing is finished.
```

Comment:
Why does the sitename needs to be removed from the snippet title after editing? I thought it should stay there.